### ISSUE TITLE
"TypeError: object of type 'float' has no len()" for -inf entropy (single atom in ORCA)

### ISSUE THREAD

**Issue by schneiderfelipe** [cclib/cclib#816]
The following comes from Travis:

```python3
    def testORCA_ORCA4_1_single_atom_freq_out(logfile):

        """ORCA frequency with single atom."""

        assert len(logfile.data.vibdisps) == 0

        assert len(logfile.data.vibfreqs) == 0

        assert len(logfile.data.vibirs) == 0

>       assert len(logfile.data.entropy) == -numpy.inf

E       TypeError: object of type 'float' has no len()

test/regression.py:1651: TypeError
```

I want to address it in #812. It is a simple fix, just summing the actual contributions for entropy.
(Giving `-inf` entropy on single atom calculations is [an issue on ORCA's side](https://orcaforum.kofo.mpg.de/viewtopic.php?f=8&t=5436).)

**Comment by langner** [comment]
Sounds good, looks like a coding mistake in the single-atom PR we merged a few days ago. Feel free to do it in a separate PR, that would be fine as well.

**Comment by langner** [comment]
Actually, it'd be better to fix this separately since it's blocking other PRs and #812 I think was waiting on a few comments still.

**Comment by schneiderfelipe** [comment]
> Actually, it'd be better to fix this separately since it's blocking other PRs and #812 I think was waiting on a few comments still.

Fine. I'll do it separately.

**Comment by langner** [comment]
> (Giving `-inf` entropy on single atom calculations is [an issue on ORCA's side](https://orcaforum.kofo.mpg.de/viewtopic.php?f=8&t=5436).)

Sorry to be such a pedant, but could you add a comment in the parser code explaining this is an ORCA issue causing us to do the summation in this case?
